pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+onlyBuiltDependencies:
```

Comment:
original project doesn't use `pnpm` to keep things simple, and your changes don't update the README to indicate it should be used.

please remove pnpm usage.